changelog.md

```diff
@@ -6,36 +6,36 @@
 
 * Refactored application CSS and updated Source Interface design (#6322, #6362, #6315, #6419, #6429)
 * Updated code to remove Flask and Jinja2 deprecation warnings (#6245)
-* Fixed cases where string formatting was breaking translation lookups (#6368)
 * Simplified templates using jinja2 expression-statement extension (#6378)
 * Updated filesystem_id fields to be non-nullable (#6350)
 * Updated gnupg commands to use direct trust model (#6397)
 * Replaced potentially offensive terms in wordlists (#6402)
-* Added validation checks for configured vs. supported languages (#6406)
+* Gracefully emit errors when configured languages are unavailable (#6406)
+* Fixed broken link to download Tor Browser on Tor2Web interstitial (#6393, #6430)
+* Add locale for Portugese (Portgual), with the language code `pt_PT` (#6156)
```

Comment:
```suggestion
* Add locale for Portuguese (Portugal), with the language code `pt_PT` (#6156)
```

Though we can also revisit this once we're sure `pt_PT` will reach coverage (as we've discussed out of band)!

Reply:
I'm so embarrassed and mostly annoyed that I spelled these wrong ._.